src/plugins/largeProfileAvatars/index.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and Board
```

Comment:
I think this needs to include contributors.

```suggestion
 * Copyright (c) 2023 Vendicated, Board, and contributors
```